divvy/compute.py

```diff
@@ -262,6 +262,7 @@ def write_script(self, output_path, extra_vars=None):
         if extra_vars:
             if not isinstance(extra_vars, list):
                 extra_vars = [extra_vars]
+            extra_vars.reverse()                
```

Comment:
Do we want the input list to actually be changed? I think the reversal will be visible after this function call.

Reply:
I was imagining they'd just be giving a list on-the-fly...but yeah, that could be a problem if they're giving a named list...